packages/components/src/internal/components/ontology/actions.ts

```diff
@@ -1,6 +1,7 @@
 import { ActionURL, Ajax, getServerContext, Utils } from '@labkey/api';
 
 import { ConceptModel, OntologyModel, PathModel } from './models';
+import * as Path from 'path';
```

Comment:
was this import intentional or maybe just added by accident? I don't see `Path` being used.

Reply:
unintentional, removed.